### PR TITLE
Datastore max size faq

### PR DESCRIPTION
**Pull request by zippeurfou**
Adding FAQ for datastore max size and include max upload time.

### CODE REVIEW

File: products/add-data-to-grid-datastores/datastore-faq.md [hunk 1]
```diff
@@ -20,6 +20,10 @@ When you create a datastore, you certify that you have the rights to that data.
 
 Grid does not charge for storing data. However, there is a limit of 25 datastores.
 
+## Does Grid have a limit on DataStore size?
+
+Yes Grid DataStore max size is 1.5 TB. In addition, for security reasons we only allow you to upload data for 24 hours. If your data is too big to upload in 24 hours you will need to manually resume the upload every 24 hours.
```
Comment (hhsecond) [inline products/add-data-to-grid-datastores/datastore-faq.md:25]: Should we rephrase it to something like this to avoid confusion about what's "manually resume" means?

```
we only allow you to upload data for 24 hours continuously. If your data is too big to upload in 24 hours, you session will break. But initiating `grid datastore create <args...>` again will resume the upload instead of starting from scratch again 
```

**Comment by zippeurfou** [comment]
@hhsecond edited according to your suggestion. Let me know your thoughts.

**Review by hhsecond — APPROVED** [review]
LGTM

**Comment by pchitale1** [comment]
Looks good, Merging

**Review by pchitale1 — COMMENTED** [review]
looks good
